bin/entrypoint

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
```

Comment:
what do you think about renaming this file to `set-env-versions`?

Reply:
works for me!

Reply:
done!